frontend/src/metabase/ui/components/utils/index.ts

```diff
@@ -3,4 +3,5 @@ export * from "./DelayGroup";
 export * from "./Divider";
 export * from "./FocusTrap";
 export * from "./Paper";
+export * from "./Rem";
```

Comment:
That's not a util "component". I'd re-export `rem` from `metabase/ui/index.ts` directly.

Reply:
Tbh, I was confused that we have util components.
This belongs to utils, but we don't have it.

To avoid introducing more confusion, I'll export it directly from index.ts as you suggested.